gpMgmt/bin/gpexpand

```diff
@@ -2052,10 +2052,20 @@ class gpexpand:
 
         conn = dbconn.connect(_gp_expand.dburl, encoding='UTF8')
         # get GUC value: gp_interconnect_proxy_addresses
-        sql = "show gp_interconnect_proxy_addresses;"
-        cursor = dbconn.execSQL(conn, sql)
-        icproxy_addresses_string = cursor.fetchone()[0].strip()
-        cursor.close()
+        try:
+            sql = "show gp_interconnect_proxy_addresses;"
+            cursor = dbconn.execSQL(conn, sql)
+            icproxy_addresses_string = cursor.fetchone()[0].strip()
+            cursor.close()
+        except Exception as e:
+            if 'unrecognized configuration parameter' in str(e):
+                self.logger.info("gp_interconnect_proxy_addresses GUC does not exist, skipping IC Proxy validation.")
+                conn.close()
+                return
+            else:
```

Comment:
After the `return`, it is pointless to use `else`.

Reply:
done